vllm/attention/ops/triton_decode_attention.py

```diff
@@ -39,11 +39,12 @@
 
 logger = logging.getLogger(__name__)
 
-# TODO: Remove this when triton>=3.2.0. This issue will not affect performance
+# Remove this warning when triton>=3.2.0. This issue will not affect performance
```

Comment:
how about: "only print the following warnings when triton version < 3.2.0. The issue won't affect performance or accuracy."

Reply:
done